content/community.html

```diff
@@ -30,6 +30,15 @@
     more elaborate or controversial changes, please discuss them on the mailing
     list before sending a pull request.
   </p>
+  <h1>Commercial support</h1>
+  <p>
+    This is a list of third-party companies which provide supportt or
+    consulting services for Prometheus. Prometheus is an independent open source
+    project which does not endorse any company.
+  </p>
+    <ul>
```

Comment:
The `<ul>` should either not be indented, or be wrapped in another `<p>` or something.